### ISSUE TITLE
I just lost important note on auto save 

### ISSUE THREAD

**Issue by primozs** [laurent22/joplin#198]
### Operating system
- [ *] macOS
### Application
- [ *] Desktop

I pressed cmd-z or cmd-t 
and the note was gone replaced by the sibling note.

where are notes stored and is it possible to get it back? this might just cost me some money

**Comment by primozs** [comment]
1. bug and autosave no way to get the note back. 
2. sqlite note body empty
3. one month of note updates gone
4. so long joplin

**Comment by laurent22** [comment]
Sorry to hear you've lost a note. Did you use Time Machine by any chance? Because then you can restore ~/.config/joplin-desktop and you would get the note back.

It's not entirely clear what series of shortcuts got the note deleted, but once it is Joplin indeed doesn't have versioning so it's not possible to get it back.

I'll check cmd-z anyway to see if there's an issue that could make it clear the content of a note.

**Comment by primozs** [comment]
@laurent22 Wow!! Thank You!! I did not thought of that! It worked! :D 

Previous version 0.10.59
1. I wanted to press ctrl-z but I press ctrl-t  this made the  note look like it was overwritten with the first sibling note. This happened for two different notes. Then I panicked. There was no way to get the note back so I stopped and upgraded Joplin. The end. 
2. After that I review content of the sqlite db and note body was empty.
3. In upgraded version 0.10.60 I can not repeat the overwrite. If I press ctrl-t the note A disappears but if i click another note B and back on the note A the content is back. Without overwrite. 

Like suggested time machine saved the day!
